app/src/main/java/io/multy/ui/adapters/AssetTransactionsAdapter.java

```diff
@@ -32,16 +44,94 @@ public Holder onCreateViewHolder(ViewGroup parent, int viewType) {
     @Override
     public void onBindViewHolder(Holder holder, int position) {
         DecimalFormat fiatFormat = new DecimalFormat("#.##");
-        TransactionHistory transactionHistory = data.get(position);
-        ArrayList<TransactionHistory.StockExchangeRate> stockRates = transactionHistory.getStockExchangeRates();
-        String amountBtc = transactionHistory.getTxOutAmount();
-        double amountFiat = stockRates != null && stockRates.size() > 0 ? stockRates.get(0).getBtcUsd() : 16.000;
-        String amountFiatString = fiatFormat.format(amountFiat) + " USD";
+        TransactionHistory transaction = transactions.get(position);
+        boolean isInput = true;
+        switch (transaction.getTxStatus()) {
```

Comment:
I guess we can simplify that as isInput = transaction.getTxStatus().equals(Constants.TX_STATUS_IN_BLOCK_CONFIRMED) || transaction.getTxStatus().equals(Constants.TX_STATUS_IN_MEMPOOL_INCOMING)